src/components/billableMetrics/BillableMetricItem.tsx

```diff
@@ -64,21 +63,23 @@ export const BillableMetricItem = memo(
           tabIndex={0}
           {...navigationProps}
         >
-          <BillableMetricName>
-            <Avatar size="big" variant="connector">
+          <div className="flex min-w-0 flex-1 items-center">
+            <Avatar className="mr-3" size="big" variant="connector">
               <Icon name="pulse" color="dark" />
             </Avatar>
-            <NameBlock>
+            <div className="mr-6 min-w-0">
               <Typography color="textSecondary" variant="bodyHl" noWrap>
                 {name}
               </Typography>
               <Typography variant="caption" noWrap>
                 {code}
               </Typography>
-            </NameBlock>
-          </BillableMetricName>
-          <CellSmall align="right">{formatTimeOrgaTZ(createdAt)}</CellSmall>
-          <ButtonMock />
+            </div>
+          </div>
+          <Typography className="mr-6 w-28" align="right">
+            {formatTimeOrgaTZ(createdAt)}
+          </Typography>
+          <div className="w-10" />
```

Comment:
Why do we need this empty div?

Reply:
The button is placed with an absolute position based on the parent relative, parent being the whole line container.

To make the items well alligned we need to "reserve" some place for the button to be placed at the end of the line, pushing away other line infor (name, code, etc)

Reply:
This is the old way we have to create those buttons, a event prevent default would do the job 🤷